Sources/Xendit/Models/DTOs/XenditCardData.swift

```diff
@@ -21,10 +21,36 @@ import Foundation
     // Card verification number (CVN) value
     open var cardCvn: String?
 
-    public init(cardNumber: String, cardExpMonth: String, cardExpYear: String) {
+    // Card holder first name value
+    open var cardHolderFirstName: String?
+
+    // Card holder first name value
+    open var cardHolderLastName: String?
+
+    // Card holder first name value
+    open var cardHolderEmail: String?
+
+    // Card holder first name value
+    open var cardHolderPhoneNumber: String?
```

Comment:
if you already created `XenditCardHolderInformation`, wouldn't it be better to have
```
open var cardHolderInformation: XenditCardHolderInformation?
```

Pro:
- Any additional fields for card holder later on only needs to update one place (`XenditCardHolderInformation`), and not both card holder and card data
- Card holder can be instantiated as a single object and reused (say multiple cards under same card holder)
Con:
- One extra layer to access information

Reply:
The API in the background puts all cardHolderInformation inside the card data object. Therefore I wanna make the experience with the API to be similar @billliu-work

Reply:
There is a difference between API layer and objects you use in your app layer (ie. you do a mapping during JSON decoding), but I guess this is just something small